packages/create-cloudflare/templates/queues/js/wrangler.toml

```
@@ -1,5 +1,5 @@
 name = "<TBD>"
-main = "src/index.js"
+main = "src/index.ts"
```

```suggestion
main = "src/index.js"
```

here we are in `/queues/js/` so I think that this is incorrect, isn't it?